src/main/java/com/myfirstownapp/beeapp/harvest/HarvestController.kt

```diff
@@ -0,0 +1,22 @@
+package com.myfirstownapp.beeapp.harvest
+
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/beeapp/harvest")
+class HarvestController (private val service:HarvestDbService){
+
+    @GetMapping(value = ["getHarvest"])
+    fun getAllHarvest(): List<Harvest> = service.getAllHarvest()
+
+    @GetMapping(value = ["getHarvest/{kind}"])
```

Comment:
It's bad pattern. It`s better to use a query param instead of the path variable.